pocs/utils/social_slack.py

```diff
@@ -0,0 +1,25 @@
+import requests
```

Comment:
I'm not familiar with the requests library. Where does that come from?

Reply:
This was in @wtgee code he originally used to output to Slack.
See the original code [here](https://github.com/panoptes/POCS/blob/may-run/pocs/core.py#L185).
I just reused it.

Reply:
It's not standard python but it's more straight-forward to use than `urllib` or `urllib2`. I was actually surprised we weren't already using it somewhere. It's a popular library so I don't have any issues including it.

Reply:
We don't have this in requirements.txt. @wtgee do you know how we are getting it installed?

Reply:
Let's put it in the requirements. We hadn't used this before so wasn't there.